connect/hwinfo.go

```diff
@@ -0,0 +1,262 @@
+package connect
+
+import (
+	"bufio"
+	"bytes"
+	"net"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var (
+	cloudEx = `(?m)Version: .*(amazon)|Manufacturer: (Amazon)|Manufacturer: (Google)|Manufacturer: (Microsoft) Corporation`
```

Comment:
I don't think it should be a multiline match.

Reply:
yeah, I'll take it out. I can't remember what I was trying

Reply:
done